File: src/test/java/com/ragin/bdd/Application.java
package com.ragin.bdd;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ApplicationContext;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan("com.ragin.bdd.cucumbertests.library")
public class Application {
    public static void main (String[] args) {
        ApplicationContext ctx = SpringApplication.run(Application.class, args);
    }
}

File: src/test/kotlin/com/ragin/bdd/Application.kt
package com.ragin.bdd

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication
@ComponentScan("com.ragin.bdd.cucumbertests.library")
open class Application : SpringBootServletInitializer()

@Suppress("SpreadOperator")
fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
